kubeadm-reset.md: add instruction about graceful shutdown (#45146)
kubernetes/website: content/en/docs/reference/setup-tools/kubeadm/kubeadm-reset.md

See the [etcd documentation](https://github.com/coreos/etcd/tree/master/etcdctl) for more information.

### Graceful kube-apiserver shutdown

> If you have your `kube-apiserver` configured with `--shutdown-delay-duration` flag, you can run the following commands (or their equivalents)

Reviewer: what does it mean "(or their equivalents"? perhaps we can remove this.

> If you have your `kube-apiserver` configured with `--shutdown-delay-duration` flag, you can run the following commands (or their equivalents)

Reviewer: ```suggestion
If you have your `kube-apiserver` configured with the `--shutdown-delay-duration` flag,
you can run the following commands
```

> to make `kube-apiserver` process shut down gracefully before running `kubeadm reset`:

Reviewer: ```suggestion
to make the `kube-apiserver` process shut down gracefully before running `kubeadm reset`:
```

Reviewer 2: ```suggestion
to attempt a graceful shutdown for the running API server Pod, before you run `kubeadm reset`:
```
(key point here is “attempt”)

```bash
yq eval -i '.spec.containers[0].command = []' /etc/kubernetes/manifests/kube-apiserver.yaml

> timeout 60 sh -c 'while pgrep kube-apiserver >/dev/null; do sleep 1; done' || true

Reviewer 2: This attempts a graceful shutdown, but doesn't guarantee it.

### Graceful kube-apiserver shutdown

If you have your `kube-apiserver` configured with the `--shutdown-delay-duration` flag,

> you can run the following commands
> to attempt a graceful shutdown for the running API server Pod, before you run `kubeadm reset`:

Reviewer: ```suggestion
you can run the following commands to attempt a graceful shutdown for the running API server Pod,
before you run `kubeadm reset`:
```
